pkgs/development/ocaml-modules/lacaml/default.nix

```
@@ -10,7 +10,7 @@ buildDunePackage rec {
 
   useDune2 = true;
```

```suggestion
  duneVersion = "2";
```
This can also likely be 3